src/wp-includes/functions.php

```diff
@@ -4299,7 +4459,8 @@ function _doing_it_wrong( $function, $message, $version ) {
 				$version = sprintf( __( '(This message was added in version %s.)' ), $version );
 			}
 			/* translators: %s: Codex URL */
-			$message .= ' ' . sprintf( __( 'Please see <a href="%s">Debugging in ClassicPress</a> for more information.' ),
+			$message .= ' ' . sprintf(
+				__( 'Please see <a href="%s">Debugging in ClassicPress</a> for more information.' ),
```

Comment:
not part of this PR, but this link text doesn't match its destination (wordpress)
It's a separate (old) issue to change all Codex links to CP pages.

Reply:
I've had a quick look and can't immediately find an appropriate link within the CP project.

I'm not averse to changing this in this PR if we can find an appropriate page to use as a link.

Reply:
Right, someone can write that article eventually and it will get changed with all the others.
Meanwhile, this link text should say "Debugging in WordPress" since that's what it is linking to.

Reply:
I agree, while we are pointing at upstream links the description should match.